gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -382,6 +378,12 @@ private boolean moreServerContent() throws IOException {
     }
   }
 
+  private void recreateStub(StatusRuntimeException e) {
```

Comment:
Can we refactor to accept `Status.Code` instead of `StatusRuntimeException` as an argument to make the method more re-usable ?

Reply:
In this class it makes sense to use `StatusRuntimeException` as a parameter, because this is what caller in both cases has, otherwise we will need to extract status code in 2 places.